### PR TITLE
Install specific major version

### PR DESCRIPTION
**Pull request by kamazee**
If a required major version is specified, wrapper passed it as a flag to the composer installer if it supports such flags.

### CODE REVIEW

**Comment by kamazee** [comment]
As this touches `ComposerWrapperParams` class that has been introduced recently, pinging @benbor: you might be interested to check it out.

**Comment by kamazee** [comment]
@benbor feel free to comment even though it's merged (I need to merge it to 1.2 to make way for other changes).

File: composer [hunk 1]
```diff
@@ -280,13 +274,40 @@ if (!class_exists('ComposerWrapper')) {
...
-                ),
+                    '--no-ansi',
+                    '--help',
+                );
+                $helpCommand = implode(' ', $helpCommandParts);
+                if ($this->supportsForceVersionFlag($helpCommand, $majorVersion)) {
+                    $majorVersionFlag = '--' . $majorVersion;
+                } else {
+                    $this->showError(
+                        "Installer doesn't allow to specify major version.\n" .
```
Comment (benbor) [inline composer:290]: @kamazee can you provide here some details, just for my information. When this situation possible? I've checked https://getcomposer.org/installer and it still supports major version specification

Comment (kamazee) [reply composer:290]: I want wrapper to continue working with a warning if the installer drops support for this flags (I assume that options come and go; nobody guarantees low long they will be there)

**Pull request merged**